apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/ExchangeDrawer/index.tsx

```diff
@@ -88,6 +100,7 @@ export default function ExchangeDrawer({ swapTransaction, exchangeRate, onComple
       });
       setError(error);
     },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
why are we ignoring those deps?

Reply:
adding these dependencies will cause further renders which will then cause an error.